code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -158,6 +213,23 @@ components:
           description: Hashed phone number. SHA-256 (in hexadecimal representation) of the mobile phone number in 'E164 with +' format
           type: string
           example: 32f67ab4e4312618b09cd23ed8ce41b13e095fe52b73b2e8da8ef49830e50dba
+    ueId:
+      type: object
+      required:
+        - phone_number
+      properties:
+        phone_number:
+          description: Mobile phone number (MSISDN) to verify. The number must be in international format.
```

Comment:
Here you can reuse:
$ref: '#/components/schemas/DevicePhoneNumber'

but it's up to you.